app/utilities/schema.py

```diff
@@ -25,6 +27,41 @@
 }
 
 
+def get_schema_path_map_for_language(language_code=DEFAULT_LANGUAGE_CODE):
+    schema_dirs = [
+        os.path.join(schema_dir, language_code) for schema_dir in SCHEMA_DIRS
+    ]
+
+    schema_files = []
+
+    for schema_dir in schema_dirs:
+        schema_files.extend(glob(f"{schema_dir}/*.json"))
+
+    return {
+        Path(schema_file).with_suffix("").name: schema_file
+        for schema_file in schema_files
+    }
+
+
+def get_schema_path_map():
+    language_map_codes = ["en", "cy", "ga", "eo"]
+
+    return {
+        language_code: get_schema_path_map_for_language(language_code)
+        for language_code in language_map_codes
+    }
+
+
+SCHEMA_PATH_MAP = get_schema_path_map()
```

Comment:
Any reason not to use `cache.memoize` on `get_schema_path_map` as done for `load_schema_from_name`?

Reply:
I'd prefer not to do this as it will impact every unit test we have which loads a schema due to it's dependency on flask_caching. I believe we can make used of improved caching methods available in 3.8 once we update. This method is only called once to generate this variable, maybe we can hold off?